src/shell/lisa_shell

```diff
@@ -93,10 +93,49 @@ fi
 echo -ne "$LISASHELL_DEFAULT"
 }
 
+function _lisa-update-all {
+echo -ne "${LISASHELL_BLUE}"
+echo -e "*** Update LISA installation"
+
+git update-index -q --refresh
+ret=$?
+if [ $ret -ne 0 ]; then
+    echo "lisa internal error: git update-index failed.  Please report it: https://github.com/ARM-software/lisa/issues"
```

Comment:
Can we split this in two lines, like you did for the following messages, i.e.:

```bash
echo "LISA internal error: git update-index failed"
echo "Please report it: https://github.com/ARM-software/lisa/issues"
```

Notice the usage of LISA (capitalized)... I would prefer to use that for whenever we refer to the acrimonious.